### PR TITLE
Changes in task update and assignment form

### PR DESCRIPTION
**Pull request by Ajeyakrishna-k**
### Issue:
- #911
- https://github.com/Real-Dev-Squad/website-status/issues/920

### Description:
This PR modifies the existing update form where a super user can update the task of the given issue.
- This adds a new input field.
- Adds Buttons to create a task.
- Changes the html structure. Moves the button from the top of the form to the bottom.
- Changes the text in the buttons.

### Dev Tested:
- [x] Yes

### Images/video of the change:

https://github.com/Real-Dev-Squad/website-status/assets/98796547/39392319-1c74-4add-981f-7a482f0ea916

### Coverage: 
Before: 
Screenshot 2023-10-18 at 9 07 57 AM

After:
Screenshot 2023-10-18 at 9 05 52 AM

### CODE REVIEW

File: src/components/issues/ActionForm.tsx [hunk 1]
```diff
@@ -106,74 +121,125 @@ const ActionForm: FC<ActionFormProps> = ({ taskId }) => {
...
-        <form>
-            {isLoading && <Loader />}
-            <button
-                className={styles.card__top__button}
-                type="submit"
-                disabled={isAssigned}
-                onClick={handleSubmit}
-            >
-                Assign Task
-            </button>
```
Comment (Ajeyakrishna-k) [inline src/components/issues/ActionForm.tsx:118]: Just moved this button and loader to the bottom and added a few styles. Rest everything remains the same.

Comment (Ajeyakrishna-k) [reply src/components/issues/ActionForm.tsx:118]: Also added a started date input field.

File: src/components/issues/Card.tsx [hunk 2]
```diff
@@ -136,7 +136,16 @@ const Card: FC<IssueCardProps> = ({ issue }) => {
...
+                    <ActionForm
+                        taskId={taskId || ''}
+                        createTask={async () => {
+                            console.log('Placeholder');
+                        }}
+                        updateTask={async () => {
+                            console.log('Placeholder');
+                        }}
+                        taskAssignee=""
+                    />
```
Comment (Ajeyakrishna-k) [inline src/components/issues/Card.tsx:148]: This is fixed in this PR: [958](https://github.com/Real-Dev-Squad/website-status/pull/958/files#diff-3911623737bbd829be31d7d883733f7e8b1045f1fe60a33366a5132f581c24acR284-R288)

File: src/components/issues/ActionForm.tsx [hunk 3]
```diff
@@ -106,74 +121,124 @@ const ActionForm: FC<ActionFormProps> = ({ taskId }) => {
...
+                    <button
+                        className={styles.card__top__button}
+                        type="submit"
+                        disabled={isLoading || !!assignee || !taskId}
+                        onClick={handleUpdateTask}
+                    >
+                        Assign Task
+                    </button>
+                )}
+            </div>
```
Comment (vinit717) [inline src/components/issues/ActionForm.tsx:241]: Please break this into a separate component as this file is very large

**Review by ajoykumardas12 — DISMISSED** [review]

File: src/components/issues/ActionForm.tsx [hunk 4]
```diff
@@ -106,74 +121,125 @@ const ActionForm: FC<ActionFormProps> = ({ taskId }) => {
...
+                            >
+                                Starts on:
+                            </label>
+                            <input
+                                name="started-on"
+                                id="started-on"
+                                className={` ${styles.assign} ${styles.input_date}`}
+                                type="date"
+                                onChange={(e) => {
+                                    console.log(e.target.value);
```
[the remark this answers is not in the document]
Comment (Ajeyakrishna-k) [reply src/components/issues/ActionForm.tsx]: Its been removed in PR #958

[Click here](https://github.com/Real-Dev-Squad/website-status/pull/958/commits/e1f9f5d99c9115a1c75c02bac768ce1d4b86f8b3#diff-0e2d1a5b18c3ea9c03fdf717ab3239c66c199f4b2aa8547d302487268ae2fdfbL56) to view the changes.

**Review by iamitprakash — DISMISSED** [review]

**Pull request merged**